Demo/Configuration.swift

```diff
@@ -27,7 +27,7 @@ internal enum ConfigurationConstants {
     
     static let shopperReference = "iOS Checkout Shopper"
 
-    static let shopperEmail = "checkoutshopperios@example.org"
+    static let shopperEmail = "checkoutShopperiOS@example.org"
```

Comment:
Does this change break the demo app somehow?

Reply:
I don't think so